test/test_heuristics.rb

```diff
@@ -303,10 +312,9 @@ def test_ncl_by_heuristics
       "#{samples_path}/Text/LIDARLite.ncl"
     ]
     assert_heuristics({
-      "NCL" => all_fixtures("Roff", "*.ncl"),
-      "XML" => all_fixtures("XML", "*.ncl"),
+      "XML" => all_fixtures("XML", "*.ncl") - ambiguous,
       "Text" => all_fixtures("Text", "*.ncl") - ambiguous,
-      nil => ambiguous
+      nil => all_fixtures("NCL", "*.ncl") + ambiguous
```

Comment:
Same as above. A comment here would help prevent contributors from copy-pasting this too quickly.

Reply:
Clarified all appearances of `nil =>`.